pkg/rpcclient/wsclient.go

```diff
@@ -801,26 +801,26 @@ func (c *WSClient) makeWsRequest(r *neorpc.Request) (*neorpc.Response, error) {
 	select {
 	case <-c.readerDone:
 		c.respLock.Unlock()
-		return nil, fmt.Errorf("%w: before registering response channel", c.closeErrOrConnLost())
+		return nil, fmt.Errorf("before registering response channel: %w", c.closeErrOrConnLost())
```

Comment:
Originally it was an attempt to improve @carpawell's debugging experience, but OK, let's revert it, I agree that it can't be parsed properly.